Makefile

```diff
@@ -60,14 +60,16 @@ endif
 
 ######################### UTILITIES/SETTINGS ###########################
 #
+VERSION = $(shell git name-rev --name-only --tags --no-undefined HEAD 2>/dev/null || echo git-`git rev-parse --short HEAD`)
+
 # install locations
 prefix ?= /usr/local
 datarootdir ?= $(prefix)/share
 datadir ?= $(datarootdir)
 mandir ?= $(datarootdir)/man/
 bindir ?= $(prefix)/bin
 libdir ?= $(prefix)/lib
-gameinstalldir ?= $(libdir)/openra
+gameinstalldir ?= $(libdir)/openra-$(VERSION)
```

Comment:
Not touching the `Makefile` at all sounds like a huge workaround. But I agree that it could be replaced by a suffix parameter which is empty by default.

Reply:
You can replace them at runtime using `make gameinstalldir=/usr/lib/openra-whatever1234`. The ?= indicates that.

Reply:
The launcher scripts and .desktop files also need to be changed.  That can't be done without touching the makefile.

Reply:
Sorry, that was probably confusing. I meant that it should not contain needless hardcoding that don't make sense for everyone else.

Reply:
We need to introduce a new variable for the other cases, so it would make more sense to use that variable here than to require the outside scripts to override both.